runtime/doc/builtin.txt

```diff
@@ -7412,6 +7412,9 @@ stdioopen({opts})			 *stdioopen()*
 
 		{opts} is a dictionary with these keys:
 		  |on_stdin| : callback invoked when stdin is written to.
+		  |on_print| : callback invoked when neovim needs to print
+			     a message, with the message (whose type is
+			     string) as sole argument.
```

Comment:
Is there a return value? What is the callback supposed to do?

Reply:
No return value. The callback doesn't have to do anything, but for example, in firenvim, it stores the message in a global list that is later on read and displayed through the browser's UI. This helps with troubleshooting as messages printed to stdout/stderr are otherwise lost (browser extensions can't read them at all).